.github/workflows/ci.yml

```
@@ -78,6 +78,10 @@ jobs:
             command: test
             # we disable default features because rust will otherwise unify them and turn on opencl in CI.
             args: --release --test gas_calibration_test --features calibration --no-default-features
+          - name: doctest
+            key: v3
+            command: test
+            args: --all --exclude fvm_conformance_tests --exclude fvm_integration_tests --exclude "*actor" --doc
```

```suggestion
            # we disable default features because rust will otherwise unify them and turn on opencl in CI.
            args: --all --no-default-features --exclude fvm_conformance_tests --exclude fvm_integration_tests --exclude "*actor" --doc
```

it seems there's good reasons to separate `fvm` tests